test/GogeDao.t.sol

```diff
@@ -46,49 +45,87 @@ contract DaoTest is Utility, Test {
             block.timestamp + 300
         );
 
+        // Deploy gogeDao
         gogeDao = new GogeDAO(address(gogeToken));
 
+        // enable trading on the v2 token and set dao address
         gogeToken.enableTrading();
         gogeToken.setGogeDao(address(gogeDao));
+
+        // Allow for polls to be created on gogeDao
+        gogeDao.toggleCreatePollEnabled();
     }
 
+
+    // ~~ Init state test ~~
+
+    /// @notice Verify initial state pf gpgeDao and gogeToken
     function test_gogeDao_init_state() public {
         assertEq(address(gogeToken), gogeDao.governanceTokenAddr());
         assertEq(gogeDao.pollNum(), 0);
     }
 
+    
```

Comment:
Why does this utility function split the `init_state` test and the other unit tests? This should be moved to the bottom of the file or after the `setUp()` function.

Reply:
Personally I just really like having init_state directly after the setUp() and anything that will be called multiple times as a helper function before any of the actual tests. Odd I know, but this is personal preference